test/workflows/workflows.js

```diff
@@ -44,19 +55,19 @@ describe('Workflows', () => {
             access_token: '1234',
             expires_in: 3600,
             token_type: 'Bearer',
-            scope: 'flow',
+            scope: ['flow', 'flow:workflows'],
         });
         nock('https://api.sandbox.checkout.com').get('/workflows').reply(401);
-        let cko = new Checkout(
+        const cko = new Checkout(
             '2p7YQ37fHiRr8O6lQAikl8enICesB1dvAJrpmE2nZfEOpxzE-J_Gho7wDy0HY9951RfdUr0vSaQCzRKP0-o5Xg',
             {
                 client: 'ack_vvzhoai466su3j3vbxb47ts5oe',
-                scope: ['flow'],
+                scope: ['flow', 'flow:workflows'],
                 environment: 'sandbox',
             }
         );
         try {
-            let workflows = await cko.workflows.getAll();
+            const workflows = await cko.workflows.getAll();
```

Comment:
Are we missing an assertion on the `workflows` here? If not, and only asserting on the error, maybe just do the following instead
```suggestion
            await cko.workflows.getAll();
```

Same for the other warnings below as well

Reply:
Thank you. Fixed